http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/GceRuntimeMonitor.scala

```diff
@@ -200,7 +200,19 @@ class GceRuntimeMonitor[F[_]: Parallel](
     runtimeAndRuntimeConfig: RuntimeAndRuntimeConfig
   )(implicit ev: Ask[F, AppContext]): F[CheckResult] = instance match {
     case None =>
-      checkAgain(monitorContext, runtimeAndRuntimeConfig, Set.empty, Some(s"Can't retrieve instance yet"))
+      nowInstant
+        .flatMap { now =>
+          if (now.toEpochMilli - monitorContext.start.toEpochMilli > 60000)
```

Comment:
Does this block catch the `ZONE_RESOURCE_POOL_EXHAUSTED` error?

Reply:
no..this error happens asynchronously...

Reply:
ah yea...that's probably `ZONE_RESOURCE_POOL_EXHAUSTED`. if you go to the user project, and check cloud logging, there should be a log message about insert failure with `ZONE_RESOURCE_POOL_EXHAUSTED`